webpack/assets/javascripts/react_app/components/hosts/PowerStatus.test.js

```diff
@@ -0,0 +1,54 @@
+jest.unmock('./PowerStatus');
+
+import React from 'react';
+import { mount } from 'enzyme';
+import PowerStatus from './PowerStatus';
+
+describe('PowerStatus', () => {
+
+  it('pending', () => {
+    const box = mount(
+      <PowerStatus
+        loadingStatus="PENDING"
+      />
+    );
+
+    expect(box.find('.spinner.spinner-xs').length).toBe(1);
+  });
+
+  it('error', () => {
+    const box = mount(
```

Comment:
In this and the 2 following tests you show that the desired power status is present. You don't show that no other power status is present.